libs/@local/internal-api-client-generator/typescript/api.ts

```diff
@@ -0,0 +1,260 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * HASH Internal API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0.0
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
```

Comment:
I don't see why not, I've committed them because that mirrors our approach in the HASH repo. @thehabbos007 was there any reasoning that I'm not aware of for committing the files?

Reply:
I've committed them for now because that mirrors the approach we took in the HASH repo, and because we don't have the Open API spec hosted anywhere (it's in the private internal API repo for now).

Down the line when we have the open API spec reliably hosted somewhere we could look into using a postinstall script.

Out of curiosity, @thehabbos007 was there any reasoning I'm not aware of for not doing this in the HASH repo for the `graphClient`?

Reply:
If anything I'd put the spec in here and auto-generate the TS files from that – updates to the API will be easier to read and less of a diff

Reply:
> we don't have the Open API spec hosted anywhere
we could serve it from the API server?

Reply:
> If you're referring to the internal API server, I agree and that's what I intend on doing. But we do not have a deployment of the internal API, so that would end up blocking this PR.

Understood, makes sense – let's leave it until we have that then